region/brinstar/green/Spore Spawn Kihunter Room.json

```diff
@@ -166,11 +120,41 @@
     {
       "from": 2,
       "to": [
-        {"id": 1}
+        {"id": 1},
+        {"id": 2}
       ]
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway",
+      "notable": false,
+      "requires": [],
```

Comment:
This requires killing the left kihunter so it should have some basic requirement.

Reply:
Done.